### ISSUE TITLE
Mobile support

### ISSUE THREAD

**Issue by Bytezz** [syncthing-gtk/syncthing-gtk#51]
Consider an UI for narrow windows and touch input support, after GTK4 port of course.  
Would enhance use on smartphones, tablets and 2-in-1 laptops.

<details> 
<summary>Current situation on mobile</summary>

![Syncthing-gtk on PinePhone](https://github.com/syncthing-gtk/syncthing-gtk/assets/25850701/e256479e-0cf1-4229-9a93-6e5ad15d9b92)

</details>

<details> 
<summary>Rough mockup</summary>

![Rough mockup](https://github.com/syncthing-gtk/syncthing-gtk/assets/25850701/a84ca926-31f8-4051-9dd0-b6c0972abd72)

</details>

**Comment by Salamandar** [comment]
I agree. Once we go through the gtk4 port we cèuld use libhandy (libadwaita) and have a dynamic layout.

**Comment by mikoducky** [comment]
woah, are you actually daily driving a linux phone?!? last I tried, it was garbage with potential

**Comment by Bytezz** [comment]
An OG PinePhone daily driven for more than 2 years. With kde connect, syncthing and flatpaks there is a real Linux ecosystem. Poor call audio quality and fast discharging battery are a problem, but (for me) it's worth it.
But about a month ago it stopped turning on completely, I don't know whats wrong.
Since then I'm back on my old Android phone and I'm hating Google OS

**Comment by Rudd-O** [comment]
I considered a Pinephone for a while but then I tried it, and it was too slow to be usable, so I decided against it.

**Comment by agx** [comment]
Besides the main view the prefs dialog is too wide as well, see screenshot in https://github.com/syncthing/syncthing/issues/9248 (libadwaita has `AdwViewSwitcher` to handle this).
